code/modules/antagonists/superheroes/buzzon/buzzon_spells.dm

```diff
@@ -0,0 +1,37 @@
+/obj/item/implant/spell/specified_type
+	var/spell_type = /obj/effect/proc_holder/spell/aoe_turf/conjure/creature/bee/buzzon
+
+/obj/item/implant/spell/specified_type/implant(mob/living/target, mob/user, silent = FALSE, force = FALSE)
+	spell = new spell_type()
+	. = ..()
+
+
+/obj/item/implant/spell/specified_type/bees
+	name = "bluespace bee storage implant"
+	desc = "Allows user to create swarms of bees on will."
```

Comment:
```suggestion
	desc = "Allows user to create swarms of bees at-will."
```
maybe add a "the" in front of "user" as well? how are the other implant descriptions worded?